src/main/java/ru/yandex/practicum/filmorate/service/InMemoryService/InMemoryFilmService.java

```diff
@@ -73,8 +73,4 @@ public List<Film> getFilms() {
         return filmStorage.getFilms();
     }
 
-    public List<Film> getTopFilms(Integer count) {
-        log.info("Get {} popular films", count);
-        return filmStorage.getTopFilms(count);
-    }
```

Comment:
Предлагаю не удалять реализацию inMemory. Ниже приведу решение совмещения одноименных методов (типа перегрузка).

```suggestion
    public List<Film> getTopFilms(Integer count) {
        log.info("Get {} popular films", count);
        return filmStorage.getTopFilms(count);
    }
```